### PR TITLE
Created home page and started individual photo page

### PR DESCRIPTION
**Pull request by redliz5808**

### CODE REVIEW

File: src/pages/Home/index.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  }
+
+  render() {
+    return (
+      <>
+        {this.state.isLoading && <Loading />}
+        <Container>
+          {this.state.data &&
+            !this.state.isLoading &&
+            Object.entries(this.state.data).map((entry) => {
```
Comment (yoloOnTheBattlefield) [inline src/pages/Home/index.js]: You can use Objec.values here :D

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Object/values

File: src/pages/Photo/index.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import axios from "axios";
+
+class Photo extends React.Component {
+  getPhoto = async (photoid) => {
+    try {
+      const { data } = await axios(
+        `https://api.unsplash.com/photos/:${photoid}&client_id=${process.env.REACT_APP_API_KEY}`
+      );
+      console.log(data);
```
Comment (yoloOnTheBattlefield) [inline src/pages/Photo/index.js]: no console

**Review by yoloOnTheBattlefield — CHANGES_REQUESTED** [review]

File: src/App.js [hunk 3]
```diff
@@ -1,25 +1,33 @@
...
+              <StyledLink to="/">Home</StyledLink>
+            </StyledLi>
+            <StyledLi>
+              <StyledLink to="/user/:userid">User</StyledLink>
+            </StyledLi>
+          </StyledUl>
+        </StyledNav>
+
+        <Switch>
+          <Route component={(props) => <Home {...props} />} exact path="/"></Route>
```
Comment (yoloOnTheBattlefield) [inline src/App.js]: why not ` <Route component={Home} exact path="/" />`? you are not passing anything to Home

File: src/pages/Home/Home.js [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+  handleClick = (button) => {
+    if (button === "Previous" && this.state.page === 1) {
+      this.setState({ page: 1 });
+    } else if (button === "Previous" && this.state.page > 1) {
+      this.setState({ page: this.state.page - 1 });
+    } else if (button === "Next") {
+      this.setState({ page: this.state.page + 1 });
+    } else {
+      this.setState({ page: button });
```
Comment (yoloOnTheBattlefield) [inline src/pages/Home/Home.js]: button will come in as a string, you might have to convert it to number

File: src/pages/Home/Home.js [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+
+  render() {
+    return (
+      <>
+        {this.state.isLoading && <Loading />}
+        {this.state.data && !this.state.isLoading && (
+          <>
+            <Container>
+              {Object.values(this.state.data).map((value) => {
+                console.log(value);
```
Comment (yoloOnTheBattlefield) [inline src/pages/Home/Home.js]: 😅😅😅

**Review by yoloOnTheBattlefield — CHANGES_REQUESTED** [review]

**Review by aOrro — APPROVED** [review]

**Pull request merged**